src/execute_request.jl

```diff
@@ -59,7 +59,7 @@ Generate the preferred MIME representation of x.
 Returns a tuple with the selected MIME type and the representation of the data
 using that MIME type.
 """
-function display_mimestring(mime_array::Vector{MIME}, x)
+function display_mimestring(mime_array::AbstractVector{MIME}, x)
```

Comment:
`AbstractVector{<:MIME}` would be broader as an argument type signature.   (However, since this is an internal function, `Vector{MIME}` is okay too.)